app/build.gradle

```diff
@@ -70,7 +70,7 @@ dependencies {
 
     // Poynt SDK and Model Dependencies
     implementation 'co.poynt.api:android-api-model:1.2.118-SNAPSHOT@jar'
-    implementation 'co.poynt.android.sdk:poynt-sdk:1.2.32-SNAPSHOT@aar'
+    implementation 'co.poynt.android.sdk:poynt-sdk:1.2.34-SNAPSHOT@aar'
```

Comment:
Please change to our Jazinda release versions:
    implementation 'co.poynt.api:android-api-model:1.2.126@jar'
   implementation 'co.poynt.android.sdk:poynt-sdk:1.2.34@aar'

Reply:
updated in new commit. please re-review. thanks!